oaknational/Oak-Web-Application
# chore: 488-move-search-to-beta

## Description

- Moves search page from `/search` to `/beta/search`

## Issue(s)

Fixes #488 

## How to test

1. Go to /search
2. See 404 page

1. Go to /beta/search
2. See search page (currently not functional)


## Checklist
